styles/styles.css

```diff
@@ -61,12 +61,15 @@
   --link-hover-color: #000;
   --background-color: white;
   --light-color: #f7f7f7;
+  --vv-dark-grey-color: #070707;
   --dark-color: #d2d2d2;
   --text-color: black;
   --orange-color: #fa6200;
   --blue-color: #0080ff;
   --green-color: #80ff00;
   --yellow-color: #fce403;
+  --pumpkin-orange-color: #ff6400;
```

Comment:
How close is this "pumkin" color to the orange? Can we combine them?

Reply:
fixed.